ext/config.m4

```diff
@@ -2,8 +2,20 @@ PHP_ARG_WITH(cassandra, Enable Cassandra extension,
 [  --with-cassandra[=DIR]    Enable the Cassandra extension.])
 
 AC_MSG_CHECKING([for supported PHP version])
-PHP_CASSANDRA_FOUND_PHP_VERSION=`${PHP_CONFIG} --version`
+if test -z "$PHP_VERSION"; then
+  if test -z "$PHP_CONFIG"; then
+    AC_MSG_ERROR([php-config not found])
+  fi
+  PHP_CASSANDRA_FOUND_PHP_VERSION=`$PHP_CONFIG --version`
+else
+  PHP_CASSANDRA_FOUND_PHP_VERSION=`echo "$PHP_VERSION"`
```

Comment:
No reason to `echo` here; simply assign the `PHP_CASSANDRA_FOUND_PHP_VERSION` variable.